src/main/scala/no/ndla/articleapi/controller/ArticleControllerV2.scala

```diff
@@ -282,13 +286,12 @@ trait ArticleControllerV2 {
       "/external_id/:deprecated_node_id",
       operation(
         apiOperation[ArticleIdV2]("getInternalIdByExternalId")
-          summary "Get id of article corresponding to specified deprecated node id"
-          description "Get internal id of article for a specified ndla_node_id"
+          summary "Get id of article corresponding to specified deprecated node id."
+          description "Get internal id of article for a specified ndla_node_id. Node_id is used as topic and resource id in ndla.no."
```

Comment:
Dette er vel ikke nødvendigvis sant for nye artikler? 
Taksonomi lager en uuid, feks: `urn:resource:5ad08d05-49aa-4e3a-b1ab-b72fd53d72a8`